nsm-nse.html

```diff
@@ -2,11 +2,11 @@
   <head>
     <meta
       name="go-import"
-      content="cisco-app-networking.github.io/nsm-nse git https://github.com/adodon2go/nsm-nse"
+      content="cisco-app-networking.github.io/nsm-nse git https://github.com/cisco-app-networking/nsm-nse"
     />
     <meta
       name="go-source"
-      content="cisco-app-networking.github.io/nsm-nse     https://github.com/adodon2go/nsm-nse https://github.com/adodon2go/nsm-nse/tree/vanity{/dir} https://github.com/adodon2go/nsm-nse/blob/vanity{/dir}/{file}#L{line}"
+      content="cisco-app-networking.github.io/nsm-nse     https://github.com/cisco-app-networking/nsm-nse https://github.com/cisco-app-networking/nsm-nse/tree/vanity{/dir} https://github.com/cisco-app-networking/nsm-nse/blob/vanity{/dir}/{file}#L{line}"
```

Comment:
I think we should change vanity-> master here. @ondrej-fabry , what do you think?